packages/stackable/stackables/express.js

```diff
@@ -0,0 +1,85 @@
+'use strict'
+
+const { request } = require('undici')
+const BaseStackable = require('./base')
+
+class ExpressStackable extends BaseStackable {
+  #url
+  #port
+  #server
+
+  constructor (options) {
+    super(options)
+    this.app = options.app
```

Comment:
This is problematic. This stackable would need to 100% wrap the express application, so it would need to load the server file, capture the app, etc. cc @ShogunPanda